cmd/gossamer/toml_config.go

```diff
@@ -54,34 +55,17 @@ func loadConfig(cfg *ctoml.Config, fp string) error {
 }
 
 // exportConfig exports a dot configuration to a toml configuration file
-func exportConfig(cfg *ctoml.Config, fp string) *os.File {
-	var (
-		newFile *os.File
-		err     error
-		raw     []byte
-	)
-
-	if raw, err = toml.Marshal(*cfg); err != nil {
-		logger.Errorf("failed to marshal configuration: %s", err)
-		os.Exit(1)
-	}
-
-	newFile, err = os.Create(filepath.Clean(fp))
+func exportConfig(cfg *ctoml.Config, targetPath string) (err error) {
```

Comment:
Shouldn't we still return the `*os.File`? I feel like a function named `exportConfig` should return a file handler.

Reply:
I would say no because:

1. this is an unexported function
2. that `*os.File` that would be returned would never be used (except for `file.Name()`)

So it just feels natural to give the config and the target file path to export to I think.